package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shift-reducer",
-  "version": "2.0.6",
+  "version": "3.0.3",
```

Comment:
Because I foolishly published a 3.0.2 when messing with the package.json. Which works, it just can't run its tests on node versions before 5.